### PR TITLE
Day7pt1 iss#01

### PR DESCRIPTION
**Pull request by Cobra915**
got it finished. 

I tried adding some code to display_output that would write the File_Tree dict to a JSON or txt documents but I couldn't get it working.

### CODE REVIEW

File: python/Test_Space.py [hunk 1]
```diff
@@ -1,37 +1,92 @@
-import os
```
Comment (Mietzsche81) [inline python/Test_Space.py:1]: Don't commit test space to your repo. These are temporary files.

File: python/day7/convert.txt [hunk 2]
```diff
@@ -0,0 +1,5 @@
+localfiles
```
Comment (Mietzsche81) [inline python/day7/convert.txt:1]: Is this example output? If so, don't commit it.

File: python/day7/day7part1/__main__.py [hunk 3]
```diff
@@ -6,53 +6,29 @@
...
-    '''
+    import build as build
 
     # Read input
     fileName = sys.argv[1]
     file = extract.parse_input(fileName)
     data = extract.process_commands(file)
     # Process Data
-    output = transform.process_data(data)
+    File_Tree = build.build_file_tree(data)
```
Comment (Mietzsche81) [inline python/day7/day7part1/__main__.py:16]: `File_Tree` -> `file_tree`

https://pep8.org/#naming-conventions

File: python/day7/day7part1/__main__.py [hunk 4]
```diff
@@ -6,53 +6,29 @@
...
+    #   if "--process-size" in sys.argv[1:]:
+    #       result = transform.process_file_size(File_Tree)
+    #   if "--process-location" in sys.argv[1:]:
+    #       transform.process_absolute_path(File_Tree)
+    #   if "--help" in sys.argv[1:]:
+    #       print(
+    #           "--process.size"
+    #           "--process-location"
+    #       )
+    R_List = transform.find_small_directories(File_Tree)
```
Comment (Mietzsche81) [inline python/day7/day7part1/__main__.py:27]: Delete commented code?

File: python/day7/day7part1/display_output.py [hunk 5]
```diff
@@ -1,9 +1,18 @@
...
+    # with open('convert.txt', 'w') as convert_file:
+    #    for key, nested in sorted(File_Tree.items()):
+    #        print(key, file=convert_file)
+    #        for subkey, value in sorted(nested.items()):
+    #            print('   {}: {}'.format(subkey, value), file=convert_file)
+    #        print(file=convert_file)
+    
 
-    pprint.pprint(display)
+    # print(f"The filesystem was written to {convert_file}")
```
Comment (Mietzsche81) [inline python/day7/day7part1/display_output.py:14]: Delete commented code

File: python/day7/day7part1/display_output.py [hunk 6]
```diff
@@ -1,9 +1,18 @@
 import pprint
+import json
```
Comment (Mietzsche81) [inline python/day7/day7part1/display_output.py:2]: Delete unused imports

**Comment by Mietzsche81** [comment]
@Cobra915 There are some binary `.pyc` files that should be deleted too.

**Review by Mietzsche81 — CHANGES_REQUESTED** [review]

**Review by Cobra915 — COMMENTED** [review]
Commented changes made

**Review by Mietzsche81 — APPROVED** [review]

**Pull request merged**